.github/instructions/patterns-templates-refactoring.instructions.md

```
@@ -0,0 +1,874 @@
+/**
+ * [Pattern Name] - [Brief description]
+ *
+ * @package    Tour_Operator
+ * @subpackage Patterns  
+ * @since      1.0.0
+ */
+
+// phpcs:ignoreFile PluginCheck.CodeAnalysis.ImageFunctions.NonEnqueuedImage
+
+return array(
+	'title'         => __( '[Pattern Name]', 'tour-operator' ),
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)